backend/src/schema/types/type/User.gql

```diff
@@ -97,14 +97,14 @@ type User {
   contributions: [Post]! @relation(name: "WROTE", direction: "OUT")
   contributionsCount: Int! @cypher(
   statement: """
-  MATCH (this)-[: WROTE]->(r: Post)
+  MATCH (this)-[:WROTE]->(r:Post)
   WHERE NOT r.deleted = true AND NOT r.disabled = true
   RETURN COUNT(r)
   """
   )
 
   comments: [Comment]! @relation(name: "WROTE", direction: "OUT")
-  commentedCount: Int! @cypher(statement: "MATCH (this)-[: WROTE]->(: Comment)-[: COMMENTS]->(p: Post) WHERE NOT p.deleted = true AND NOT p.disabled = true RETURN COUNT(DISTINCT(p))")
+  commentedCount: Int! @cypher(statement: "MATCH (this)-[:WROTE]->(:Comment)-[:COMMENTS]->(p:Post) WHERE NOT p.deleted = true AND NOT p.disabled = true RETURN COUNT(DISTINCT(p))")
```

Comment:
Does anybody know why we're not checking `WHERE NOT comment.deleted = true AND NOT comment.disabled = true` here?

Reply:
I understand that we count the posts commented by the user and not the comments themselves. So I think the cypher statement would be more complex. We should look for all the comments by the user that are nether disabled nor deleted and then count the posts commented by these comments.

Reply:
Does this work:
```cypher
MATCH (this)-[:WROTE]->(c:Comment)-[:COMMENTS]->(p:Post)
WHERE NOT p.deleted = true AND NOT p.disabled = true
WHERE NOT c.deleted = true AND NOT c.disabled = true
RETURN COUNT(DISTINCT(p))
```

Reply:
Exactly that would work. If you comment a post and disable the comment, that would first increase and decrease the `commentedCount`. Whereas without the extra condition, the `commentedCount` would stay the same.

Now that we're talking about it, I remember that we distinguish between `commentedCount` and `commentsCount`. One is the number of posts commented, the other is the number of comments written.

The `DISTINCT` in the query above will make sure that `commentedCount` does not increase if you comment the same post twice.

Reply:
I'm going to leave this as it is though.